tests/test_magnification_dialog.py

```diff
@@ -8,28 +8,31 @@
 
 
 @pytest.mark.parametrize(
-    "front_fiducial1, front_fiducial2, back_fiducial1, back_fiducial2, expected_magnification_params",
+    "front_fiducial1, front_fiducial2, back_fiducial1, back_fiducial2, expected_magnification_params, double_click",
```

Comment:
You could double-parametrize here. Then less fiddly handling of cases:
```python
@pytest.mark.parametrize("click_twice", [True, False])
@pytest.mark.parametrize("fiducials....", ...)
```

Reply:
Thanks! I didnt know this was possible.